src/leiningen/github_cdn.clj

```diff
@@ -63,23 +66,23 @@
           (println "Removing old files")
           (git "rm" "-rf" "*" {:dir *dir* :throw false})
           (copy-files src-dir *dir*)
-          (println "Commiting and pushing")
+          (println "Committing and pushing")
           (git "add" "*" {:dir *dir* :throw false})
           (git "commit" "-m" comment' {:dir *dir* :throw false})
           (git "push" "origin" (str branch ":" branch) {:dir *dir*})
           (println "Done")))
       (println "No source dir specified"))))
 
 (defn github-cdn
-  "Publishes resources to gihhub branch"
+  "Publishes resources to gh-pages branch"
```

Comment:
Yeah sorry for not being clear - what I meant was using the branch var for all output to stdout.  As you say, using the branch name in commit messages is a bit nonsensical.